muikku-timed-notifications-plugin/src/main/java/fi/otavanopisto/muikku/plugins/timed/notifications/strategies/AssessmentRequestNotificationStrategy.java

```diff
@@ -175,7 +175,7 @@ private List<SchoolDataIdentifier> getStudentIdentifiersWithoutEvaluationActivit
         continue;
       }
       
-      Date studyStartDate = getDateResult(result.get("studyStartDate"));
+      Date studyStartDate = getStudyStartDateIncludingTemporaryLeaves(result);
```

Comment:
Since all (?) notifiers check `studyStartDate` being `null`, shouldn't those checks also skip the student if `studyStartDate` is in the future?